trade-finance-manager-ui/component-tests/utilisation-reports/_macros/keying-sheet-table.component-test.ts

```diff
@@ -110,7 +110,7 @@ describe(component, () => {
       const wrapper = getWrapper({ ...aKeyingSheetTableViewModel(), userCanEdit });
 
       wrapper.expectElement('thead input#select-all-checkbox').toExist();
-      wrapper.expectElement('thead th:has(input#select-all-checkbox)').toHaveAttribute('rowspan', '2');
+      wrapper.expectElement('thead td:has(input#select-all-checkbox)').toHaveAttribute('rowspan', '2');
```

Comment:
If scope col is being added (see other comment), would be good to have a test here too: `.toHaveAttribute('scope', 'col');`

Reply:
Other conversation resolved - keeping as is